### ISSUE TITLE
Can not render button in IE intranet mode

### ISSUE THREAD

**Issue by lepatachou** [paypal/paypal-checkout#567]
Hi, I have the problem with Win7 / IE 11 (11.0.9600.18860) after applied all solutions describe in [this post](https://github.com/paypal/paypal-checkout/issues/395). I'm using "client side REST" script from https://developer.paypal.com/demo/checkout/#/pattern/client and the PayPal button is not displayed.

I tried to open the html file from my HD or from my website in https.  In developpment tools, I see :
SCRIPT5022: Can not render button in IE intranet mode
Fichier : checkout.js, ligne : 8841, colonne : 21

Thanks for your help.

**Comment by lepatachou** [comment]
After some test, works with 
    <script src="https://www.paypalobjects.com/api/checkout.min.js"></script>
instead of
    <script src="https://www.paypalobjects.com/api/checkout.js"></script>

**Comment by bluepnume** [comment]
OK, glad that works.

**Comment by kulkarnisan** [comment]
Hi guys, 
I tried with using checkout.min.js file ...its not working for me. 
By manually doing the setting in IE 11, as mentioned in the other post[https://github.com/paypal/paypal-checkout/issues/395](url) the button is visible though....but
is there anyway not to ask the end user to do this kind of setting to use the paypal button on IE 11? I got to know about registry setting from the same post but still I am not sure how my client will react about that.
thanks
